lib/pacemaker/pcmk_sched_clone.c

```diff
@@ -369,19 +369,24 @@ pcmk__clone_action_flags(pe_action_t *action, const pe_node_t *node)
     return pcmk__collective_action_flags(action, action->rsc->children, node);
 }
 
+/*!
+ * \internal
+ * \brief Apply a location constraint to a clone resource's allowed node scores
+ *
+ * \param[in,out] rsc       Clone resource to apply constraint to
+ * \param[in,out] location  Location constraint to apply
```

Comment:
The variable is still named `constraint`

Reply:
Thanks